zitadel/v2/default_login_policy/funcs.go

```diff
@@ -0,0 +1,206 @@
+package default_login_policy
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/zitadel/zitadel-go/v2/pkg/client/zitadel/admin"
+	"github.com/zitadel/zitadel-go/v2/pkg/client/zitadel/policy"
+	"google.golang.org/protobuf/types/known/durationpb"
+
+	"github.com/zitadel/terraform-provider-zitadel/zitadel/v2/helper"
+)
+
+func delete(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	tflog.Info(ctx, "default login policy cannot be deleted")
+	return nil
+}
+
+func update(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	tflog.Info(ctx, "started update")
+
+	clientinfo, ok := m.(*helper.ClientInfo)
+	if !ok {
+		return diag.Errorf("failed to get client")
+	}
+
+	client, err := helper.GetAdminClient(clientinfo)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	current, err := client.GetLoginPolicy(ctx, &admin.GetLoginPolicyRequest{})
+	if err != nil {
+		return diag.FromErr(err)
+	}
```

Comment:
Why don't we compare to the TF state?

Reply:
compare what to the TF state?

Reply:
To the declared state. Comparision is made here: https://github.com/zitadel/terraform-provider-zitadel/pull/36/files#diff-c3f8681c64238c3b31799bf985066776ec5d64f242fc39cabe4bae2fb1bf97e9R71